app/assets/stylesheets/vendor/autoSuggest.css

```diff
@@ -95,24 +93,18 @@ ul.as-selections li.as-selection-item a:active.as-close {
 }
 
 ul.as-selections li.as-original {
-	margin-left: 0;
+  width: 100%;
+	margin: 0;
   list-style: none;
+  padding: 0;
 }
 
 ul.as-selections li.as-original input {
-	border: none;
 	outline: none;
 	font-size: 13px;
-	width: auto;
-	padding: 0;
-  	margin: 0;
-	height: 20px;
+  margin: 0;
 	line-height: 20px;
-	width: 300px;
-}
-
-ul.as-selections li.as-original.as-original{
-	width: auto;
+	width: 100%;
```

Comment:
Actually please make sure to normalize indentation, hard tap here, I think we decided on two or four spaces?